core/commands/repo.go

```diff
@@ -226,13 +224,13 @@ lockfiles, as well as the api file. This command can only run when no ipfs
 daemons are running.
 `,
 	},
-	Run: func(req oldcmds.Request, res oldcmds.Response) {
-		configRoot := req.InvocContext().ConfigRoot
+	Run: func(req *cmds.Request, res cmds.ResponseEmitter, env cmds.Environment) error {
+		cctx := env.(*oldcmds.Context)
+		configRoot := cctx.ConfigRoot
```

Comment:
This should use the new `ConfigRoot` helper function just merged in https://github.com/ipfs/go-ipfs/pull/5712.